### ISSUE TITLE
[A11Y] UNav Header Translate lacks locale, needs Dropdown

### ISSUE THREAD

**Issue by mendeza** [ITS-HCD/nysds#1412]
The UNav Header has a "Translate" subcomponent with a list of available locales.

1. The component interaction lacks keyboard and screen reader affordances: refactor to use DS Dropdown
2. The locale items lack the `lang` attribute and appropriate values, providing a sub-par screen reader experience: update each item accordingly.
